client/components/upgrades/google-apps/dialog/users.jsx

```diff
@@ -57,6 +57,7 @@ var GoogleAppsUsers = React.createClass( {
 	},
 
 	inputsForUser: function( user, index ) {
+		const contactText = this.translate( 'contact', { context: 'part of e-mail address', comment: 'As it would be part of an e-mail address contact@example.com' } );
 		var domain = this.props.domain;
```

Comment:
`domain` can be made `const` and merged into one `const` statement along with `contactText`.